README.md

```diff
@@ -1,81 +1,70 @@
-## Synchronisation & Encryption
-#### Synchronisation
-The sync process consists in having two actors, one is the ClientServer, which exists on a Virtual Machine and the second one being the CentralServer.
+## Synchronisation and Encryption
 
-##### Setup ClientServer
-The fallowing script needs to be run on TAO instance in order to make a client server.
+### Synchronisation
+The synchronization process is based upon two actors, the Client Server, which is installed on a Virtual Machine, and the Central Server.
+
+#### Setting up the Client Server
+The following script needs to be run on a TAO instance in order to create a Client Server:
 
 ```php
 sudo -u www-data php index.php '\oat\taoOffline\scripts\tools\setup\SetupClientServer'
 ```
-    In case taoEncryption it's installed this script it's gonna setup the instance with encryption.
+
+On systems where [extension-tao-encryption](https://github.com/oat-sa/extension-tao-encryption) is installed, the script will set it up with encryption.
     
- In order to point this instance to a specific server the fallowing command needs to be run
+Point the instance to a specific server by executing the following command:
  
  ```php
-    sudo -u www-data php index.php '\oat\taoSync\scripts\tool\RegisterHandShakeRootURL' --rootUrl=http://tao-central.dev/
+sudo -u www-data php index.php '\oat\taoSync\scripts\tool\RegisterHandShakeRootURL' --rootUrl=http://tao-central.dev/
  ```
- 
- 
-##### Setup CentralServer
-The fallowing script needs to be run on TAO instance in order to make a central server.
+
+#### Setting up the Central Server
+Run the following to turn a TAO instance into a Central Server.
 
 ```php
 sudo -u www-data php index.php 'oat\taoOffline\scripts\tools\setup\SetupCentralServer'
 ```
-     In case taoEncryption it's installed this script it's gonna setup the instance with encryption.
 
-##### Types of Syncs available
-* Test Center Based on OrganisationId
+Again, instances with `taoEncryption` will benefit from encryption.
+
+#### Types of available synchrounizations
+* Test Center Based on the `OrganisationId`
     * Users:
         * Test Takers
         * Proctor Administrators
         * Proctor
         * Eligibility
         * Deliveries
```

Comment:
These two instances are not `Users`.
It may be also useful to mention types of synchronizations per direction (VM - > SC or CS -> VM). From Central Server to VM we synchronize Test Centers, Users, Deliveries, Eligibilities and LTI Consumers. From VM to the Central Server we synchronize Test Sessions, Results, Result Logs and LTI Users.

Reply:
done